docs/ref/forms/widgets.txt

```diff
@@ -569,6 +569,30 @@ These widgets make use of the HTML elements ``input`` and ``textarea``.
     * ``template_name``: ``'django/forms/widgets/search.html'``
     * Renders as: ``<input type="search" ...>``
 
+``TelInput``
+~~~~~~~~~~~~~~
+
+.. versionadded:: 5.2
+
+.. class:: TelInput
+
+    * ``input_type``: ``'tel'``
+    * ``template_name``: ``'django/forms/widgets/tel.html'``
+    * Renders as: ``<input type="tel" ...>``
+
+    The input value is not automatically validated to a particular format
+    before the form can be submitted, because formats for telephone numbers
+    vary so much around the world.
+
+    Client-side validations can be added by setting ``pattern``, ``minlength``
+    and ``maxlength`` keys on the :attr:`Widget.attrs` argument.
```

Comment:
Suggested edits for clarity and grammatical correctness (“validations” shouldn't be plural 😉)

```suggestion
    The input value has no client-side validation by default because telephone
    number formats vary so much around the world. You can add some by setting
    ``pattern``, ``minlength``, or ``maxlength`` in the :attr:`Widget.attrs`
    argument.
```